src/utils/register-foreign-multiple.coffee

```diff
@@ -0,0 +1,65 @@
+program = require 'commander'
+msgflo_nodejs = require 'msgflo-nodejs'
+fs = require 'fs'
+path = require 'path'
+foreigner = require '../foreign-participant'
```

Comment:
Noe that this is not really needed to have a `Participant` instance just to send a discovery message. Can also us `msgfloNodejs.Client.registerParticipant(definition)`